rocAL/include/loaders/image/node_cifar10_loader.h

```diff
@@ -39,7 +39,8 @@ class Cifar10LoaderNode : public Node {
     /// \param load_batch_count Defines the quantum count of the images to be loaded. It's usually equal to the user's batch size.
     /// The loader will repeat images if necessary to be able to have images in multiples of the load_batch_count,
     /// for example if there are 10 images in the dataset and load_batch_count is 3, the loader repeats 2 images as if there are 12 images available.
-    void init(const std::string &source_path, const std::string &json_path, StorageType storage_type, bool loop, size_t load_batch_count, RocalMemType mem_type, const std::string &file_prefix);
+    void init(const std::string &source_path, const std::string &json_path, StorageType storage_type, bool loop, size_t load_batch_count, RocalMemType mem_type, const std::string &file_prefix,
+              RocalBatchPolicy _last_batch_policy = RocalBatchPolicy::PARTIAL, bool last_batch_padded = false);
```

Comment:
Have we tested the CIFAR10 with last batch?

Reply:
I think its better to remove it from cifar10. 

@shobana-mcw please add your thoughts.

Reply:
Removed CIFAR10 loader changes